### DISCUSSION TITLE
Where is version 3..2.1 and chunkedhtml

### DISCUSSION THREAD

**Discussion by myfta** [jgm/pandoc#9960] [General]
I have downloaded version 3.2.1 for Windows (msi) and installed it onto a machine that previously had version 2.17.1.1.
I was trying to use the chunkedhtml feature and the command returned this error:
`Unknown output format chunkedhtml`

That prompted me to check the version:
```
D:\Downloads>pandoc --version
pandoc 2.17.1.1
Compiled with pandoc-types 1.22.1, texmath 0.12.4, skylighting 0.12.2,
citeproc 0.6.0.1, ipynb 0.2
```

So what is going on here?

If I run `pandoc --help` it does not show **chunkedhtml**

Has the version just not been changed in the code?
Has the upgrade (overwrite) not worked as it should, or does the repo not contain the correct file?

What is the correct sequence of syntax to split an html into individual files, splitting on Heading 1? I tried:

```
D:\Downloads>pandoc -t chunkedhtml --split-level=1 -o RH_File RH_Test.html
Unknown option --split-level.
```

**Comment by jgm** [comment]
Most likely the old version is in your PATH before the new version.  Try uninstalling the old version.

**Reply by myfta** [reply]
I had uninstalled (and cleaned up with Revo) - then I discovered there was another old copy in my Downloads folder. Thank you.

I would still appreciate a steer on the chunking task - thanks.

**Reply by jgm** [reply]
It should work once you're running the right version.
